src/prefect/infrastructure/provisioners/container_instance.py

```diff
@@ -768,6 +768,62 @@ async def _aci_credentials_block_exists(
         except ObjectNotFound:
             return False
 
+    async def _create_provision_table(self, work_pool_name: str, client: PrefectClient):
+        return Panel(
+            dedent(
+                f"""\
+                    Provisioning infrastructure for your work pool [blue]{work_pool_name}[/] will require:
+
+                        Updates in subscription: [blue]{self._subscription_name}[/]
+
+                            - Create a resource group in location: [blue]{self._location}[/]
+                            - Create an app registration in Azure AD: [blue]{self.APP_REGISTRATION_NAME}[/]
+                            - Create/use a service principal for app registration
+                            - Generate a secret for app registration
+                            - Create an Azure Container Registry with prefix [blue]{self.REGISTRY_NAME_PREFIX}[/]
+                            - Create an identity [blue]{self._identity_name}[/] to allow access to the created registry
+                            - Assign Contributor role to service account
+                            - Create an ACR registry for image hosting
+                            - Create an identity for Azure Container Instance to allow access to the registry
+
+                        Updates in Prefect workspace
+
+                            - Create Azure Container Instance credentials block: [blue]{self._credentials_block_name}[/]
+                    """
+            ),
+            expand=False,
+        )
+
+    async def _customize_resource_names(
+        self, work_pool_name: str, client: PrefectClient
+    ) -> bool:
+        self.RESOURCE_GROUP_NAME = prompt(
```

Comment:
If we're updating these names, I think we should lowercase them. The upper case suggests that they're a constant.

Reply:
Yeah that makes sense, their values are no longer constant 😥

Reply:
Applied in [1093d01](https://github.com/PrefectHQ/prefect/pull/11412/commits/1093d017fd3f2fdd197ed517fc12ffeb6de78df8)